docs/types/response.md

```diff
@@ -3,42 +3,110 @@
 
 @signature `response(status, body, headers, statusText)`
 
-Used to detail a response.
+  Used to detail a response.
 
-Example:
+  Example:
 
-```js
-fixture( { url: "/todos/{action}" },
-	function( request, response, headers, ajaxSettings ) {
-		response(
-			401,
-			{ message: "Unauthorized" },
-			{ "WWW-Authenticate": "Basic realm=\"myRealm\"" },
-			"unauthorized" );
-	}
-);
+  ```js
+  import {fixture} from "can";
+  import "//unpkg.com/jquery@3.3.1/dist/jquery.js";
 
-$.post( "/todos/delete" );
-```
+  fixture( {url: "/todos/{action}"},
+    ( request, response, headers, ajaxSettings ) => {
+      response(
+        401,
+        { message: "Unauthorized" },
+        { "WWW-Authenticate": "Basic realm=\"myRealm\"" },
+        "unauthorized"
+      );
+    }
+  );
 
-You don't have to provide every argument to `response`. It can be called like:
+  $.post( "/todos/delete" ).then( result => {
+    console.log( JSON.parse(result) ); //-> {
+    //   headers: {WWW-Authenticate: "Basic realm='myRealm'"},
+    //   responseBody: {message: "Unauthorized"},
+    //   statusCode: 401,
+    //   statusText: "unauthorized"
+    // }
+  } );
+  ```
+  @codepen
 
-```js
-// Just body
-response( { message: "Hello World" } );
+  You don't have to provide every argument to `response`. It can be called like:
 
-// status and body
-response( 401, { message: "Unauthorized" } );
+  ```js
+  import {fixture} from "can";
+  import "//unpkg.com/jquery@3.3.1/dist/jquery.js";
 
-// body and headers
-response( "{\"message\":\"Unauthorized\"}", { "WWW-Authenticate": "Basic realm=\"myRealm\"" } );
+  fixture( {url: "/todos"}, ( request, response ) => {
+    // Just body
+    response( { message: "Hello World" } );
+  });
 
-// status, body statusText
-response( 401, "{\"message\":\"Unauthorized\"}", "unauthorized" );
-```
+  $.get("/todos").then( result => {
+    console.log( result ); //-> "{'message':'Hello World'}"
+  } );
 
-The default `statusText` will be `ok` for `200 <= status < 300, status === 304` and `error`
-for everything else.
+  ```
+  @codepen
+  @highlight 6,only
```

Comment:
When you get to this page, lets talk about how it can be re-arranged.  I feel like we should create multiple signatures.

Reply:
My proposal for how this should go is in 162c8fa. Move the other examples to the _Use_ section.

Reply:
Updated based on previous conversation to include all examples in separate signatures in 68154d2.